src/audio/speech_to_text/stt_calculator.cc

```diff
@@ -24,25 +24,21 @@
 #pragma GCC diagnostic pop
 #pragma warning(pop)
 
-#include "../../http_payload.hpp"
-#include "../../logging.hpp"
+#include "src/http_payload.hpp"
+#include "src/logging.hpp"
 #include <mutex>
 #include <thread>
+#include <chrono>
 
 #pragma warning(push)
 #pragma warning(disable : 6001 4324 6385 6386)
 #include "absl/strings/escaping.h"
 #include "absl/strings/str_cat.h"
 #pragma warning(pop)
+#define DR_WAV_IMPLEMENTATION
```

Comment:
Is it intended to be always defined? If yest we could move this to src/port/dr_audio.
Or it could be defined as local_defines in src/port/BUILD for dr lib